src/License/Migrations/RefreshLicensesForLastActiveDate.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Give\License\Migrations;
+
+use Give\Framework\Migrations\Contracts\Migration;
+
+/**
+ * @unreleased
+ */
+class RefreshLicensesForLastActiveDate extends Migration
+{
+    /**
+     * @unreleased
+     */
+    public static function id(): string
+    {
+        return 'refresh-licenses-for-last-active-date';
+    }
+
+    /**
+     * @unreleased
+     */
+    public static function title(): string
+    {
+        return 'Refresh Licenses to set Last Active license date';
+    }
+
+    /**
+     * @unreleased
+     */
+    public static function timestamp(): int
+    {
+        return strtotime('2025-01-15 00:00:00');
```

Comment:
Let's make sure this date is up to date (today).  The migration timestamp is important because it determines the order of the migrations being run.